config/i18n/all-langs.js

```diff
@@ -0,0 +1,247 @@
+'use strict';
```

Comment:
What is this file?

Reply:
It should be getting ignored by git - not sure why it isn't here. It's generated from the `all-langs.ts` file in the same folder.

Reply:
yes, It's generated from the all-langs.ts file

Reply:
It should be removed from this PR

Reply:
removed please merge